Utilities/src/test/java/edu/stanford/hivdb/mutations/MutationSetTest.java

```diff
@@ -638,4 +687,29 @@ public void testHashCode() {
 			new MutationSet(Gene.RT, "31X")
 		);
 	}
-}
+	
+	@Test 
+	public void testHasSharedMutation() {
+		final Mutation mut1 = new Mutation(Gene.RT, 68, "A");
+		final Mutation mut2 = new Mutation(Gene.RT, 115, "FR");
+		final Mutation mut3 = new Mutation(Gene.RT, 118, "I");
+		final MutationSet muts = new MutationSet(mut1, mut2, mut3);
+		assertTrue(muts.hasSharedAAMutation(mut1));
+		assertTrue(muts.hasSharedAAMutation(mut2));
+		assertTrue(muts.hasSharedAAMutation(mut3));
```

Comment:
Can you add this line here:

`assertTrue(muts.hasSharedAAMutation(new Mutation(Gene.RT, 115, "F")));`